draft-private-access-tokens.md

```diff
@@ -129,21 +129,7 @@ the available content it can serve based on the client's geographical region.
 Origins almost exclusively use client IP addresses for this purpose.
 
 
-## Client Linking
-
-An origin wishes to link multiple accesses from a client. The server does not
```

Comment:
Can we mention this in privacy considerations, and say that a Client replaying tokens (and origins accepting that) would allow this happen again.

Reply:
I'll address this in a followup PR.